.gitignore

```diff
@@ -10,4 +10,6 @@ obj/
 /WebAssets/dist
 /WebAssets/coverage
 /Owin.Demo/wwwroot
-/AspNetCore.Demo/wwwroot
\ No newline at end of file
+/AspNetCore.Demo/wwwroot
+/AspNetCore.Demo/dist

```

Comment:
This one is surprising -- there should not be a `dist` under AspNetCore.Demo I think.
Can't check yet due to also running into #130, but I'll double-check after that.

Reply:
Hmm, I can't repro. It may have been created when I was debugging the build. I removed the line.